packages/node_modules/@ciscospark/redux-module-media/src/helpers.js

```diff
@@ -51,7 +51,7 @@ function constructCallStatus(call) {
       isInitiated = false;
       isConnected = false;
     }
-    isIncoming = call.direction === 'in' && !call.hasJoinedOnThisDevice;
+    isIncoming = call.direction === 'in' && !call.joinedOnThisDevice;
```

Comment:
`hasJoinedOnThisDevice` is a property that we calculate below. `joinedOnThisDevice` comes directly from the sdk's call object.